javathreads/src/main/java/bootcamp/Threads.java

```diff
@@ -0,0 +1,111 @@
+package bootcamp;
+
+import java.util.UUID;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Threads {
+
+    private static int executionNumber;
+
+    public static void main(String[] args) {
+
+        //Initialising the executionNumber
+        executionNumber = 0;
+
+        String message = "Execution time of main method: ";
+
+        // Check the time at the start of the execution
+        long startTime = System.currentTimeMillis();
+
+        System.out.println("Let's run the programm without threads!");
+
+        // Calling the procedure 10 times
+        timeConsuming(3);
```

Comment:
vairāk strādās ar šo metodi; time consuming būs kā palīgs; to neviens vairāk nemainīs

šeit gribu
time
print 1
time 
print a
....
time 
print 4
time
print e